doc/comparison.md

```diff
@@ -0,0 +1,25 @@
+# Comparison
+
+| Tool    | Binary | Source Code/LLVM | Available Under-constrained Algorithm |
+|---------|--------|------------------|---------------------------------------|
+| [angr](https://angr.io/)    | Yes    | No*              | Yes*                                   |
+| [Grease](https://github.com/GaloisInc/grease)  | Yes    | Yes              | Yes                                   |
```

Comment:
Spell this as "GREASE" for consistency with the rest of the documentation (and similarly for other occurrences of "Grease" in this document).